### PR TITLE
Modified so all license info from license table retrieved

### PR DESCRIPTION
**Pull request by terrasea2**
PEP8 as well

Trello Card: https://trello.com/c/4St94WPZ/2624-create-additional-copyright-type

### CODE REVIEW

File: ojssqla/ojs.py [hunk 1]
```diff
@@ -602,6 +602,7 @@ class Licenses(Base):
     id = Column(BigInteger, primary_key=True)
     name = Column(String(200))
     pretty_name = Column(String(200))
+    description = Column(String(200))
```
Comment (fradeve) [inline ojssqla/ojs.py:605]: Is this the only change introduced in this PR? If so, we should really start using different commits for changes and PEP8.

Comment (terrasea2) [reply ojssqla/ojs.py:605]: Yes it is, but I'd introduced PEP8 changes earlier to try and make sense of the code. I forgot they were there and couldn't be bothered removing them. They do no harm

Comment (terrasea2) [reply ojssqla/ojs.py:605]: I need this for the other PR to work https://bitbucket.org/ubiquitypress/glenlivet/pull-requests/170/if-license-name-ends-in-us-will-use-values/diff

**Comment by fradeve** [comment]
LGTM :+1:

**Review by fradeve — APPROVED** [review]

**Pull request merged**